### ISSUE TITLE
Some keyboard shortcuts ignored in Safari/vscode.dev

### ISSUE THREAD

**Issue by daviddossett** [microsoft/vscode#149478]
I'm able to use shortcuts to do things like open the command palette, but vscode.dev ignores others like `⌘,` to open settings. Instead, macOS captures the shortcut and executes it as if I wasn't using it in the vscode.dev context.

## Safari

https://user-images.githubusercontent.com/25163139/168323407-b33b2fee-95bd-4161-a9db-0a1312204207.mp4

## Edge

Most of these seem to work fine with the exection of [⌘K ⌘T] to switch themes. Edge still opens up a new tab.

https://user-images.githubusercontent.com/25163139/168323607-b065d450-02d7-4e99-8927-ab3426cc9677.mp4

**Comment by alexdima** [comment]
This is controlled by each feature author. We need to change the defaults we register to the keybinding registry:
- [ ] we should not use `cmd+,` on Safari as the default keybinding for settings @sandy081 
- [ ] we should not use `cmd+k cmd+t` on Edge as the default keybinding for changing themes @aeschli

**Comment by sandy081** [comment]
I am not sure if we would like to change these shortcuts per platform, because as a vscode user I am used to these keybindings. It also applies to cmd+p to open files but in browser it opens print dialog.

**Comment by daviddossett** [comment]
Agreed that they shouldn't be changed. It seems like different browsers will hijack (or rather, we don't successfully hijack) different keybindings.
- Safari brings up its own preferences instead of the Settings page on `cmd ,`
- Edge (both macOS and Windows) will open a new tab instead of opening the theme quick pick

@sandy081 I'm able to use cmd+p or ctrl+p to open files without getting the print dialog.

**Comment by sandy081** [comment]
> I'm able to use cmd+p or ctrl+p to open files without getting the print dialog.

But I can't 😞

**Comment by daviddossett** [comment]
Hm, strange. I tried on both macOS and Windows across various browsers. Regardless, it's a problem I think we should solve especially since we suggest shortcuts (e.g. cmd + k cmd + t) that don't actually work.

**Comment by RosemaryOrchard** [comment]
> we suggest shortcuts (e.g. cmd + k cmd + t) that don't actually work.
This for me is the biggest problem, I then have to find the menu commands for these actions which I don't necessarily know the name of but do know the keyboard shortcuts of.

**Comment by bhavyaus** [comment]
Perhaps we should consider not showing the shortcuts in the menu which won't work. 
<img width="233" alt="image" src="https://user-images.githubusercontent.com/25044782/236301284-e119ba0f-50e8-4e20-9697-6dcd984624f0.png">
